grpc/server/src/main/java/io/helidon/grpc/server/SslConfiguration.java

```diff
@@ -0,0 +1,201 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.grpc.server;
+
+import java.io.File;
+import java.net.URL;
+
+import io.helidon.config.Config;
+
+public class SslConfiguration {
+
+    /**
+     * True if use jdk ssl implementation
+     */
+    private boolean jdkSSL;
+
+    /**
+     * The TLS certs file.
+     */
+    private String tlsCert;
+
+    /**
+     * The TLS key file.
+     */
+    private String tlsKey;
+
+    /**
+     * The TLS CA file.
+     */
+    private String tlsCaCert;
+
+    /**
+     * Create a new instance.
+     */
+    public SslConfiguration(boolean jdkSSL, String tlsCert, String tlsKey, String tlsCaCert){
+        this.jdkSSL = jdkSSL;
+        this.tlsCert = tlsCert;
+        this.tlsKey = tlsKey;
+        this.tlsCaCert = tlsCaCert;
+    }
+
+    /**
+     * Return true if use jdk ssl implementation
+     */
+    public boolean isJdkSSL(){
+        return jdkSSL;
+    }
+
+    /**
+     * Return TLS certs file.
+     *
+     * @return the TLS certs file
+     */
+    public String getTLSCerts() {
+        return tlsCert;
+    }
+
+    /**
+     * Return the TLS key file.
+     *
+     * @return the location of the TLS key file to use
+     */
+    public String getTLSKey() {
+        return tlsKey;
+    }
+
+    /**
+     * Return the TLS CA certs file.
+     *
+     * @return the TLS CA certs file
+     */
+    public String getTLSClientCerts() {
+        return tlsCaCert;
+    }
+
+    /**
+     * Return an instance of builder.
+     *
+     * @return  an instance of builder
+     */
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    /**
+     * Return an instance of builder based on the specified external config.
+     *
+     * @param config external config
+     * @return an instance of builder
+     */
+    public static Builder builder(Config config) {
+        return new Builder(config);
+    }
+
+    /**
+     * Create an instance of sslConfig from external configuration source.
+     *
+     * @param config external config
+     * @return an instance of sslconfig
+     */
+    public static SslConfiguration create(Config config) {
+        return builder(config).build();
+    }
+
+    /**
+     * Builds the configuration for ssl.
+     */
+    static class Builder implements io.helidon.common.Builder<SslConfiguration>{
+
+        private boolean jdkSSL;
+
+        private String tlsCert = null;
+
+        private String tlsKey = null;
+
+        private String tlsCaCert = null;
+
+        private Builder() {};
+
+        private Builder(Config config) {
+            if (config == null){
+                return;
+            }
+
+            String path = config.get("path").asString().orElse(null);
+            File resourcesDirectory = new File(path);
```

Comment:
This should really be `Path resourcesDirectory = Paths.get(path)`. If the path is not specified in the config, we should probably default it to the current working directory, which can be easily done by changing `orElse(null)` above to `orElse(".")`